Sources/SwiftParser/Attributes.swift

```diff
@@ -332,14 +332,44 @@ extension Parser {
         )
       )
     case nil:
+      let isAttached = self.peek().isAttachedKeyword
       return parseAttribute(argumentMode: .customAttribute) { parser in
-        let arguments = parser.parseArgumentListElements(pattern: .none)
+        let arguments: [RawTupleExprElementSyntax]
+        if isAttached {
+          arguments = parser.parseAttachedArguments()
+        } else {
+          arguments = parser.parseArgumentListElements(pattern: .none)
+        }
+
```

Comment:
Instead of matching the string contents of the current token here, it’s much clearer to add `attached` as a contextual keyword. I’m fixing that in https://github.com/apple/swift-syntax/pull/1881.

Reply:
Thank you!